source/deployment/elasticsearch.rst

```diff
@@ -26,6 +26,10 @@ Elasticsearch v5.x, v6.x, and v7.x are supported.
 .. note::
     From Mattermost v5.26, you can filter inactive users, search by user role, and also search for terms inside links. This update introduces a breaking change which affects the "from" part of the search. To avoid this, reindex your Elasticsearch instance/cluster prior to upgrading.
 
+
+.. note::
+    If your installation has a large number of posts (typically in excess of 1 million but not strictly defined) the reindexing progress percentage may stay at 99% for a long time.
```

Comment:
```suggestion
    If your deployment has a large number of posts (typically in excess of 1 million but not strictly defined) the reindexing progress percentage may stay at 99% for a long time.
```

Reply:
One suggestion for this is to place it after the set up steps, so that there's more context (based on the assumption that the reader has set Elasticsearch up and the problem may affect them). Let me know what you think. :)

Reply:
Well I agree to you , yes I was actually thinking about placing it after the setup steps but wasn't sure if that was correct.

Reply:
I totally understand! I think it fits well in either place but I'm more inclined for it to be after the set up steps. :)